### PR TITLE
Webpack Bundle Analyzer

### PR DESCRIPTION
**Pull request by cea2aj**
Add the Webpack Bundle Analyzer which can be enabled by setting the environment variable ANALYZE_BUNDLE=true

J=none
TEST=manual

### CODE REVIEW

File: static/webpack-config.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const HtmlPlugin = require('html-webpack-plugin');
 const RemovePlugin = require('remove-files-webpack-plugin');
 const { merge } = require('webpack-merge');
 const { parse } = require('comment-json');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```
Comment (oshi97) [inline static/webpack-config.js:9]: this is probably not going to change much, but if you move the require into the if statement does the build speed for regular builds change at all?

Comment (cea2aj) [reply static/webpack-config.js:9]: that probably would help out a little

File: static/webpack-config.js [hunk 2]
```diff
@@ -66,6 +67,14 @@ module.exports = function () {
     })
   ];
 
+  const analyzeBundle = (process.env || {}).ANALYZE_BUNDLE === 'true';
+
+  if (analyzeBundle) {
+    plugins.push(new BundleAnalyzerPlugin({
+      analyzerMode: 'static',
```
Comment (oshi97) [inline static/webpack-config.js:74]: oh shoot I missed this the first time around, but did you say that the server mode is better? (I've only tried the static mode)

Comment (cea2aj) [reply static/webpack-config.js:74]: I'm pretty sure the server mode just serves the same HTML page that you get with the static. I was having some issues with the server mode but static worked for me

File: static/webpack-config.js (hunk 1 above)
Comment (oshi97) [reply static/webpack-config.js:9]: ok I'm down to do that then!

**Comment by cea2aj** [comment]
Closing because we decided not to include this functionality with the theme out of the box

**Pull request closed without merging**